src/navbar/navbar.styles.tsx

```diff
@@ -33,7 +33,8 @@ export const Wrapper = styled.div<StyleProps>`
     left: 0;
     right: 0;
     width: 100%;
-    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+    border-bottom: ${Border["width-010"]} ${Border["solid"]} ${Colour["border"]};
+    box-shadow: 0 2px 8px rgba(from ${Colour["bg-inverse-subtle"]} r g b / 16%);
```

Comment:
visually, this looks odd in dark mode, it feels more like a white glow. are shadows common in dark modes?